compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1676,6 +1676,9 @@ error.invalid.read.only.class.inclusion.in.non.read.only.class=\
 error.invalid.field.in.object.constructor.expr.with.readonly.reference=\
   invalid field in an object constructor expression with a ''readonly'' reference: ''{0}'' can never be ''readonly''
 
+error.mismatching.visibility.qualifiers.in.object.field=\
+  mismatching visibility qualifiers in referenced object field and referencing object field ''{0}''
```

Shall we change this to something like
```suggestion
  mismatched visibility qualifiers for field ''{0}'' in the including object and the included object
```